src/ngx_lua_resty_lmdb_module.h

```diff
@@ -50,4 +51,11 @@ extern ngx_module_t ngx_lua_resty_lmdb_module;
 #endif
 
 
+#define	CHACHA_KEY_SIZE	32
+#define CHACHA_CTR_SIZE	16
+#define CHACHA_BLK_SIZE	64
+#define POLY1305_BLOCK_SIZE	16
+#define AES_256_GCM_BLOCK_SIZE 16
```

Comment:
use https://www.openssl.org/docs/man1.1.1/man3/EVP_CIPHER_block_size.html to get block size instead of harc dode